apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -161,11 +192,10 @@ protected override DbTransaction DbTransaction
             set => _transaction = (SpannerTransaction) value;
         }
 
-        /// <summary>
-        /// </summary>
-        internal SpannerCommandTextBuilder SpannerCommandTextBuilder { get; set; }
+        private SpannerCommandTextBuilder SpannerCommandTextBuilder { get; set; }
 
         /// <summary>
+        /// Returns a copy of this <see cref="SpannerCommand"/>
         /// </summary>
         /// <returns></returns>
```

Comment:
I'll include this in my next submission which includes the "null" info for parameters.